react/features/base/util/openURLInBrowser.native.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import { Linking } from 'react-native';
+
+/**
+ * Opens URL in the browser.
+ *
+ * @param {string} url - The URL to be opened.
+ * @returns {void}
+ */
+export function openURLInBrowser(url: string) {
+    // eslint-disable-next-line no-empty-function
+    Linking.openURL(url).catch(() => { });
```

Comment:
Please don't silence errors, return the promise instead, of at the very least log the error.

Reply:
Fixed!